GooglePlayInstant/Editor/PlayInstantPublisher.cs

```diff
@@ -48,16 +48,27 @@ public static void Build()
                 return;
             }
 
+            ZipApk(baseApkPath, zipFilePath);
+        }
+
+        /// <summary>
```

Comment:
Please move this method to a new class. Perhaps "ZipUtils" ?

(I will be adding unzip later.)

Reply:
Resolved